addon/components/detail.js

```diff
@@ -376,18 +381,23 @@ export default Component.extend(SpreadMixin, HookMixin, PropTypeMixin, {
         const nonIndexDep = cellConfig.dependsOn.replace(/\.\d+/g, '.[]')
         addMetaProperty(cellConfig, '__dependsOn__', bunsenId.replace(nonIndexModel, nonIndexDep))
       }
+
+      subModel = getSubModel(bunsenModel, cellConfig.model, cellConfig.dependsOn) || {}
     }
 
+    let cellKey = index
     if (!cellConfig.children || cellConfig.children.length === 0) {
       // only add bunsenIds to leaf nodes
       addMetaProperty(cellConfig, '__bunsenId__', bunsenId)
+      cellKey = JSON.stringify(cellConfig) + JSON.stringify(subModel)
```

Comment:
`JSON.stringify` is used as a hash to uniquely identify a sub schema from another. Testing has shown that arranging different layouts with the same views resulted in no re-rendering and outperforms using the cell index as the key. `JSON.stringify` is only used on leaf cells and the view config be should relatively small in comparison to the entire form schemas. model schemas on average is smaller than the view config so even if a large model is referenced, it's not enormous. With these constraints I was comfortable with the performance of `JSON.stringify` as it doesn't add much to the cost.

Reply:
This will miss ordering though `JSON.stringify({a: '2', b: 2}) === JSON.stringify({b: 2, a: '2'}) // false`. What are the consequences of that, just losing efficiency?

Reply:
When the ordering is different, it usually means that the schema author had redefined the schema using conditions or some other means. In those cases, it's okay for the cell to be re-rendered.

Reply:
The re-rendering I'm trying to avoid is those cells that didn't undergo change.